src/rubintv/handlers/external/endpoints.py

```diff
@@ -191,13 +195,16 @@ async def update_todays_table(request: web.Request) -> web.Response:
             bucket, camera, recent_events, the_date
         )
         events = flatten_events_dict_into_list(camera, events_dict)
-        grid_columns = build_grid_columns_css(camera.channels)
+
+        metadata_url = get_metadata_url(bucket.name, camera.slug, date_str)
+        metadata_json = requests.get(metadata_url).text
```

Comment:
What happens if this file doesn't exist yet (or the `get()` fails for some reason)? I _hope_ it will always exist, but it might not at first, so there's a potential race condition here, which this needs to survive.

Reply:
Tested and this is fine.